src/foam/core/script/ScriptRunnerDAO.js

```diff
@@ -23,6 +23,11 @@ foam.CLASS({
     public ScriptRunnerDAO(DAO delegate) {
       setDelegate(delegate);
     }
+
+    public ScriptRunnerDAO(DAO delegate, String agencyName) {
```

Comment:
Where is this being called from? 
threadpoolname/agencyname should just be a property and configured by easydao/cspec like everything else.

Reply:
ScriptRunnerDAO decorator is being used by many DAOs.

It uses "threadPool" as the default agency so exposing it via a constructor to give option for the DAO to use a different agency.